src/ssl.c

```diff
@@ -17053,13 +17073,20 @@ unsigned long wolfSSL_set_options(WOLFSSL* ssl, unsigned long op)
 }
 
 
-unsigned long wolfSSL_get_options(const WOLFSSL* ssl)
+long wolfSSL_get_options(const WOLFSSL* ssl)
 {
     WOLFSSL_ENTER("wolfSSL_get_options");
 
     return ssl->options.mask;
 }
 
+long wolfSSL_clear_options(WOLFSSL* ssl, long opt)
+{
+    WOLFSSL_ENTER("SSL_clear_options");
+    ssl->options.mask &= ~opt;
```

Comment:
We should check "ssl" against NULL before dereferencing.

Reply:
Done

Reply:
I don't see where a ssl NULL check was added in this function (wolfSSL_clear_options()).